dspace-api/src/main/java/org/dspace/versioning/VersionDAO.java

```diff
@@ -87,7 +88,10 @@ public VersionImpl findByItemId(Context context, int itemId) {
     public List<Version> findByVersionHistory(Context context, int versionHistoryId) {
         TableRowIterator tri = null;
         try {
-            tri = DatabaseManager.query(context, "SELECT * FROM " + TABLE_NAME + " where " + HISTORY_ID + "=" + versionHistoryId + " order by " + VERSION_NUMBER + " desc");
+            tri = DatabaseManager.query(context, "SELECT * FROM " + TABLE_NAME 
+                    + " where " + HISTORY_ID + "=" + versionHistoryId 
```

Comment:
versionHistoryId should have been a query parameter all along, not concatenated into the query.